### DISCUSSION TITLE
Notification policies for Prometheus rules

### DISCUSSION THREAD

**Discussion by SimoneFalzone** [grafana/grafana#39145] [Alerting]
In our company we tried out the awesome Grafana 8 alerting feature. We noticed that alerts defined in Prometheus are shown properly but we can't interact with those alerts. So, we digged into the code and found out that the Prometheus alerts are not stored and only called and displayed in the frontend. 
So, our question is, do you plan to make it possible to at least create notification policies (e.g. Teams or Slack notifications) for Prometheus alerts? Or do you even plan to support Prometheus Alertmanager as a data source for alerts like cortex (to also enable silencing)?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
